Gems/Atom/Feature/Common/Assets/Shaders/PostProcessing/WhiteBalance.azsl

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <scenesrg.srgi>
+
+ShaderResourceGroup PassSrg : SRG_PerPass
+{
+    Texture2D<float4> m_inputColor;
+    RWTexture2D<float4> m_outputColor;
+
+    // Must match the struct in WhiteBalancePass.cpp
+    struct Constants
+    {
+        uint2 m_outputSize; // texture size of output
+        float2 m_outputCenter; // center of image in pixel coords
+        float m_temperature; // Color temperature. Higher values result in a warmer color temperature and lower values result in a colder color temperature.
+        float m_tint; // Factor for compensate for a green or magenta tint
```

Comment:
Recommend adding this at the end of the struct (after m_tint) for safety.
[[pad_to(16)]]